SPECS/createrepo_c/createrepo_c.spec

```diff
@@ -39,59 +38,77 @@ Requires:   %{name} = %{version}-%{release}
 headers and libraries for createrepo_c
 
 %prep
-%setup -q
+%autosetup -p1
 sed -e '/find_package(GTHREAD2/ s/^#*/#/' -i CMakeLists.txt
 sed -i 's|g_thread_init|//g_thread_init|'  src/createrepo_c.c
 sed -i 's|g_thread_init|//g_thread_init|'  src/mergerepo_c.c
 sed -i 's|g_thread_init|//g_thread_init|'  src/modifyrepo_c.c
 sed -i 's|g_thread_init|//g_thread_init|'  src/sqliterepo_c.c
 
 %build
-mkdir build && cd build
-cmake -DCMAKE_INSTALL_PREFIX:PATH=/usr ..
-make %{?_smp_mflags}
+mkdir build
+pushd build
+%cmake -DWITH_ZCHUNK=OFF -DWITH_LIBMODULEMD=OFF ..
+%make_build
+popd
 
 %install
-cd build
-make install DESTDIR=%{buildroot}
+pushd build
+%make_install
+popd
+
 ln -sf %{_bindir}/createrepo_c %{buildroot}%{_bindir}/createrepo
 ln -sf %{_bindir}/mergerepo_c %{buildroot}%{_bindir}/mergerepo
 ln -sf %{_bindir}/modifyrepo_c %{buildroot}%{_bindir}/modifyrepo
 
-%clean
-[ "$RPM_BUILD_ROOT" != "/" ] && rm -rf $RPM_BUILD_ROOT
-
+%check
+pushd build
+%make_build tests
+%make_build test
+popd
 
 %files
 %defattr(-, root, root)
 %license COPYING
-/etc/bash_completion.d/createrepo_c.bash
+%{_sysconfdir}/bash_completion.d/createrepo_c.bash
 %{_bindir}/*
-%{_lib64dir}/*.so.*
+%{_libdir}/*.so.*
```

Comment:
Could we add so version numbers to the files list?

Reply:
Same with other so files in other specs

Reply:
Fixed for all specs, thanks